etcdutl/snapshot/v3_snapshot_test.go

```diff
@@ -118,6 +118,106 @@ func TestSnapshotStatusNegativeRevisionSub(t *testing.T) {
 	require.ErrorContains(t, err, "negative revision")
 }
 
+// TestSnapshotStatusDuplicateKeys tests if TotalKey in status remains unchanged when inserting the same key multiple times
+func TestSnapshotStatusDuplicateKeys(t *testing.T) {
```

Comment:
All the test cases should be able to be consolidated into one table-driven sub tests. But not a big problem.